onnxruntime/python/tools/quantization/equalization_example/equalization.py

```diff
@@ -0,0 +1,589 @@
+
+            new_c1_w = onnx.numpy_helper.from_array(new_c1_w).raw_data
+            new_c1_b = onnx.numpy_helper.from_array(new_c1_b).raw_data
+            new_b1_w = onnx.numpy_helper.from_array(new_b1_w).raw_data
+            new_b1_b = onnx.numpy_helper.from_array(new_b1_b).raw_data
+            new_c2_w = onnx.numpy_helper.from_array(new_c2_w).raw_data
+
+            model.graph.initializer[c1_w_index].raw_data = new_c1_w
+            model.graph.initializer[c1_b_index].raw_data = new_c1_b
+            model.graph.initializer[b1_w_index].raw_data = new_b1_w
+            model.graph.initializer[b1_b_index].raw_data = new_b1_b
+            model.graph.initializer[c2_w_index].raw_data = new_c2_w
+
+        elif node_1_conv_flag and node_2_bn_flag and node_3_relu_flag and node_4_conv_flag: 
+
+            c1_w_name = model.graph.node[i].name + "." + "W"
+            c1_b_name = model.graph.node[i].name + "." + "B"
+            b1_w_name = model.graph.node[i+1].name + "." + "scale"
+            b1_b_name = model.graph.node[i+1].name + "." + "B"
+            c2_w_name = model.graph.node[i+3].name + "." + "W"
+            
+            c1_w_index = init_lookup[c1_w_name]
+            c1_b_index = init_lookup[c1_b_name]
+            b1_w_index = init_lookup[b1_w_name]
+            b1_b_index = init_lookup[b1_b_name]
+            c2_w_index = init_lookup[c2_w_name]
+
+            c1_w = model.graph.initializer[c1_w_index]
+            c1_b = model.graph.initializer[c1_b_index]
+            b1_w = model.graph.initializer[b1_w_index]
+            b1_b = model.graph.initializer[b1_b_index]
+            c2_w = model.graph.initializer[c2_w_index]
+
+            c1_w = onnx.numpy_helper.to_array(c1_w)
+            c1_b = onnx.numpy_helper.to_array(c1_b)
+            b1_w = onnx.numpy_helper.to_array(b1_w)
+            b1_b = onnx.numpy_helper.to_array(b1_b)
+            c2_w = onnx.numpy_helper.to_array(c2_w)
+
+            new_c1_w, new_c2_w, new_c1_b, new_b1_w, new_b1_b, S = layer_equalization(c1_w, c2_w, c1_b, bn_W1=b1_w, bn_b1=b1_b)
+
+            new_c1_w = onnx.numpy_helper.from_array(new_c1_w).raw_data
+            new_c1_b = onnx.numpy_helper.from_array(new_c1_b).raw_data
+            new_b1_w = onnx.numpy_helper.from_array(new_b1_w).raw_data
+            new_b1_b = onnx.numpy_helper.from_array(new_b1_b).raw_data
+            new_c2_w = onnx.numpy_helper.from_array(new_c2_w).raw_data
+
+            model.graph.initializer[c1_w_index].raw_data = new_c1_w
+            model.graph.initializer[c1_b_index].raw_data = new_c1_b
+            model.graph.initializer[b1_w_index].raw_data = new_b1_w
+            model.graph.initializer[b1_b_index].raw_data = new_b1_b
+            model.graph.initializer[c2_w_index].raw_data = new_c2_w
+
+        else:
+            continue
+          
+    return model
+
+
+def equalize(model_path, save=False, save_path="equalized_model.onnx",
+             first_input_name = "input", final_output_name = "output"):
```

Comment:
could you please also make it able to run standalone, like python equalization.py --model *

Reply:
Thank you for taking the time to review my work! I'll update the code to enable your suggested usage.